app/src/main/java/org/oppia/android/app/utility/ClickableAreasImage.kt

```diff
@@ -142,14 +118,47 @@ class ClickableAreasImage(
     }
   }
 
-  private fun showOrHideRegion(newView: View, clickableArea: ImageWithRegions.LabeledRegion) {
+  /**
+   * Called to show or hide selected image region
+   *
+   * @param clickableArea a clickable image region which we want to show or hide
+   */
+  fun showOrHideRegion(clickableArea: ImageWithRegions.LabeledRegion) {
```

Comment:
Since this is public, I suggest changing its name to "toggleRegion".

Reply:
Updated